master/buildbot/process/buildstep.py

```diff
@@ -871,6 +872,9 @@ def startCommand(self, cmd, errorMessages=[]):
 
         # stdio is the first log
         self.stdio_log = stdio_log = self.addLog("stdio")
+        # enable timestamps if requested
+        self.stdio_log.setTimestampsMode(self.timestamp_stdio)
```

Comment:
could we rename self.timestamp_stdio -> prepend_timestamps ? it is call that way inside the function and use that way in the test? 
other than that everything looks good.